giskard/ml_worker/websocket/utils.py

```diff
@@ -158,7 +158,24 @@ def map_dataset_process_function_meta_ws(callable_type):
     }
 
 
-def parse_function_arguments(client: Optional[GiskardClient], request_arguments: List[websocket.FuncArgument]):
+def _get_or_load(loaded_artifacts: Dict[str, Dict[str, Any]], type: str, uuid: str, load_fn: Callable[[], Any]) -> Any:
+    if type not in loaded_artifacts:
+        loaded_artifacts[type] = dict()
```

Comment:
This could be avoided by giving a [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict) instead of a dict, with dict as default factory